tq42/dataset.py

```diff
@@ -39,6 +39,10 @@ def __init__(
         else:
             self.data = self._get()
 
+    def __repr__(self):
+        created_at = str(self.data.created_at).replace("\n", " ")
+        return f"(Dataset ID: { self.data.id}, Name: { self.data.name}, Project ID: { self.data.project_id}, Sensitivity: { self.data.sensitivity}, Created by: { self.data.created_by}, Created at: {created_at })"
```

Comment:
doing it like this feels really brittle. Do you go in and manually adjust all of the `__repr__` functions when we actually add new fields?

Reply:
done it using dynamic data trasformation approach